project/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+from controls.models import Element
+from controls.oscal import CatalogData
+from siteapp.models import User
+
+
+# TODO "PackageRename" rename class to Project once the old Project class in /siteapp/models has been removed
```

Comment:
Instead of leaving a TODO in the file, should we just go ahead and create a clean up ticket in JIRA?

Reply:
There will definitely be a ticket in JIRA for this switchover... I added the TODO(s) to help search for those places in code when we do that ticket, but if we feel like it's messy to have in there right now, I can definitely take it out

Reply:
Have strong and loosely held opinions about TODO comments in code but I know that's a normal topic of debate and nitpicky.